themes/vuetober/src/components/Search.vue

```diff
@@ -58,9 +78,43 @@ export default {
       return this.getTeachers(this.searchTeacher);
     },
   },
+  methods: {
+    selectTeacher(teacher) {
+      this.$store.dispatch('selectTeacher', teacher);
+      this.isActive = teacher.id;
```

Comment:
Inac vlastne ked uz v globalnom store mas `selectedTeacher` tak tuto to uz nepotrebujes nie?